source/tests/system/visa_session_tests.cpp

```diff
@@ -95,6 +95,15 @@ TEST_F(VisaSessionTest, OpenWithErrorFromDriver_ReturnsDriverErrorWithUserErrorM
                                         kInvalidRsrc, kVisaErrorInstrumentDescriptorNotFoundMessage);
 }
 
+TEST_F(VisaSessionTest, OpenWithErrorFromDriver_ReturnsDriverErrorWithResourceDescriptor)
+{
+  EXPECT_THROW_DRIVER_ERROR_WITH_SUBSTR({
```

Comment:
Nit: I think this formatting is correct with our clang-format file, but an alternative I personally think reads better would be
```cpp
EXPECT_THROW_DRIVER_ERROR_WITH_SUBSTR(
      {
        visa::OpenResponse init_response;
        call_open(kVisaTestInvalidInstrumentDescriptor, visa::LOCK_STATE_NO_LOCK, "", 0, &init_response);
      },
      kInvalidRsrc,
      kVisaTestInvalidInstrumentDescriptor);
```
Same in the test directly above but this is minor and preference so up to you.

Reply:
Agreed.